durabletask/internal/shared.py

```diff
@@ -18,13 +20,15 @@ def get_default_host_address() -> str:
     return "localhost:4001"
 
 
-def get_grpc_channel(host_address: Union[str, None]) -> grpc.Channel:
+def get_grpc_channel(host_address: Union[str, None], metadata: List[Tuple[str, str]]) -> grpc.Channel:
```

Comment:
Should this be optional?
```suggestion
def get_grpc_channel(host_address: Union[str, None], metadata: Union[List[Tuple[str, str]], None]) -> grpc.Channel:
```

Reply:
yeah, made it optional